js/front/dataset/details-modal.vue

```diff
@@ -57,6 +57,14 @@ export default {
     props: {
         dataset: Object
     },
-    components: {Modal}
+    components: {Modal},
+    methods: {
+        stringify(value) {
+            if (typeof value === 'string' || value instanceof String) {
+                return value;
+            }
```

Comment:
We can have floats and integers if I remember. Did you try testing if it's an object?

It seems it's not pretty to write the check https://stackoverflow.com/a/16608045